dev-docs/source/dynamic_models.rst

```diff
@@ -31,6 +31,28 @@ It builds a Django model based on the imported schema data.
     :class:`~schematools.contrib.django.models.DatasetTable` and
     :class:`~schematools.contrib.django.models.DatasetField` models.
 
+Loading Schemas
+---------------
+
+To play with the examples, load the schema's first.
+
+The schema files are imported by ``manage.py import_schemas``.
+This command reads all available schemas from a schema repository
+(:ref:`SCHEMA_URL <SCHEMA_URL>`, default: https://schemas.data.amsterdam.nl/datasets/)
+and updates the metadata tables accordingly.
+
+All schema data is saved in the :class:`~schematools.contrib.django.models.Dataset`
+model from the :mod:`schematools.contrib.django` package.
+Upon startup, DSO-API reads all available dataset schema's from
+the :class:`~schematools.contrib.django.models.Dataset` model to constructs the models.
```

Comment:
typo: `the :class:`~schematools.contrib.django.models.Dataset` model to _construct_ the models.`